ios/SecuritySdk/Classes/ASN.1/ASN1BitString.swift

```diff
@@ -16,22 +16,23 @@ public struct ASN1BitString: ASN1Type, DEREncodable {
     
     // MARK: - Initialization
     
-    public init(_ rawValue: Data, _ type: Type = Type.none) throws {
-        self.octets = try Self.encode(rawValue, .bitString(type))
+    public init(_ rawValue: Data, _ type: Type = Type.none, ignorePadding: Bool = false) throws {
+        self.octets = try Self.encode(rawValue, .bitString(type, ignorePadding))
```

Comment:
Is ignorePadding related to ASN1 or it was a todo of DER?

Reply:
For digital signatures, it appears that compacting the bit string by ignoring the trailing zeroes in the last byte cause the signature check to fail on openssl. It might be prepending the padding rather than appending when the bit string is decoded. So I introduced this secondary variable in order to carry the context on whether to ignore the padding. This is only set to true for encoding the digital signature as a bitstring.